### PR TITLE
gh-411  added snackbar everywhere and spacing refactor

### PR DESCRIPTION
**Pull request by DmitryBespalov**
Now snackbar is visible everywhere

### CODE REVIEW

File: Multisig/UI/App/MainTabView.swift [hunk 1]
```diff
@@ -59,11 +58,10 @@ struct MainTabView: View {
         }
         .onAppear {
             // this adjusts the snack bar position
-            App.shared.snackbar.bottomEdgeSpacing =
-                BottomTabViewMetrics.tabBarHeight + self.tabBarSpacing
+            App.shared.snackbar.pushBottomPadding(ScreenMetrics.aboveTabBar)
         }
         .onDisappear {
-            App.shared.snackbar.bottomEdgeSpacing = 0
+            App.shared.snackbar.popBottomPadding()
```
Comment (MouazAlzahabi) [inline Multisig/UI/App/MainTabView.swift]: I think Set/Reset is better that push/pop

File: Multisig/UI/App/SceneDelegate.swift [hunk 2]
```diff
@@ -11,7 +11,7 @@ import SwiftUI
 
 class SceneDelegate: UIResponder, UIWindowSceneDelegate {
     var window: UIWindow?
-    
+
```
Comment (MouazAlzahabi) [inline Multisig/UI/App/SceneDelegate.swift:14]: Do we have extra line here ?

Comment (DmitryBespalov) [reply Multisig/UI/App/SceneDelegate.swift:14]: there was a "tab" character

File: Multisig/UI/Onboarding/TermsView.swift [hunk 3]
```diff
@@ -15,9 +15,9 @@ struct TermsView: View {
     @State private var showPrivacyPolicy = false
     @State private var showTerms = false
 
-    private let topPadding: CGFloat = 24
-    private let bottomPadding: CGFloat = 20
-    let interItemSpacing: CGFloat = 12
+    private let topPadding: CGFloat = Spacing.XL
```
Comment (MouazAlzahabi) [inline Multisig/UI/Onboarding/TermsView.swift]: Can we please change the short names "XL" "S" "L"

File: Multisig/UI/App/MainTabView.swift (hunk 1 above)
Comment (DmitryBespalov) [reply Multisig/UI/App/MainTabView.swift]: ok

File: Multisig/UI/UI Library/Overlays/Snackbar/SnackbarHostView.swift [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 30.07.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct SnackbarHostView<Content: View>: View {
+
```
Comment (MouazAlzahabi) [inline Multisig/UI/UI Library/Overlays/Snackbar/SnackbarHostView.swift]: Extra line

File: Multisig/UI/Onboarding/TermsView.swift (hunk 3 above)
Comment (DmitryBespalov) [reply Multisig/UI/Onboarding/TermsView.swift]: I do not understand what you mean

File: Multisig/UI/App/MainTabView.swift [hunk 5]
```diff
@@ -59,11 +58,10 @@ struct MainTabView: View {
         }
         .onAppear {
```
Comment (MouazAlzahabi) [inline Multisig/UI/App/MainTabView.swift]: Can we move this OnAppear/Disappear handlers to hostSnackbar modifier ?

**Review by MouazAlzahabi — APPROVED** [review]

**Review by MouazAlzahabi — APPROVED** [review]

**Pull request merged**